tests/components/abode/test_init.py

```diff
@@ -77,7 +77,10 @@ async def test_invalid_credentials(hass: HomeAssistant) -> None:
         ),
     ), patch(
         "homeassistant.components.abode.config_flow.AbodeFlowHandler.async_step_reauth",
```

Comment:
We shouldn't patch this. Better to let it create a flow and check that there's a flow in progress with reauth source.

Reply:
I agree, but this is not the PR where that should be fixed

Reply:
Yes, it was just a side note.